openedx/features/caliper_tracking/base_transformer.py

```diff
@@ -37,7 +37,7 @@ def add_generic_fields(self):
             'edx_event_type': self.event.get('event_type'),
             'host': self.event.get('host'),
             'session': self.event.get('session'),
-            'referrer': self.event.get('referer'),
+            'referer': self.event.get('referer'),
```

Comment:
according to [caliper doc](https://www.imsglobal.org/sites/default/files/caliper/v1p1/caliper-spec-v1p1/caliper-spec-v1p1.html#event) it is `referrer`

Reply:
Okay. But it is `referer` in edx events.